apps/designer/app/canvas/features/text-editor/text-editor.tsx

```diff
@@ -42,12 +54,32 @@ export const TextEditor = ({
   contentEditable,
   onChange,
 }: TextEditorProps) => {
+  const [italicClassName] = useState(() => nanoid());
+
+  // initially instance styles are applied to all nodes
+  // so not necessary to use layout effect
+  useEffect(() => {
+    const engine = createCssEngine();
+    engine.addPlaintextRule(`
+      .${italicClassName} { font-style: italic; }
+    `);
+    engine.render();
+    return () => {
+      engine.unmount();
+    };
+  }, [italicClassName]);
+
   // store references separately because lexical nodes
   // cannot store custom data
   // Map<nodeKey, Instance>
   const [refs] = useState<Refs>(() => new Map());
   const initialConfig = {
     namespace: "WsTextEditor",
+    theme: {
+      text: {
+        italic: italicClassName,
```

Comment:
we only need italic?

Reply:
Yep subscript/superscript will be fixed logically by disabling the other